### ISSUE TITLE
PNNX transform single softmax problem (missing input)

### ISSUE THREAD

**Issue by wizardk** [Tencent/ncnn#3918]
## context | 编译/运行环境 | バックグラウンド

Ubuntu 18.04 + NCNN 20220420 + libTorch 1.8.1

## how to reproduce | 复现步骤 | 再現方法
1.
```
    class PNNX_softmax(nn.Module):
        def __init__(self):
            super().__init__()

        def forward(self, attn):
            attn = F.softmax(attn, dim=-1)
            return attn
    m = PNNX_softmax()
    m.eval()
    m = torch.jit.script(m)
    m.save('PNNX_softmax.jit')
```

2.
```
./pnnx PNNX_softmax.jit inputshape=[8,10,10] inputshape2=[8,20,20]
```

3. There is no input
```
7767517
4 5
pnnx.Expression          pnnx_expr_1              0 1 0
Split                    splitncnn_0              1 2 0 1 2
aten::__is__             pnnx_10                  2 1 1 1 3
prim::If                 pnnx_11                  1 1 3 out0
```

**Comment by nihui** [comment]
https://github.com/Tencent/ncnn/tree/master/tools/pnnx#usage
export torchscript via trace but not script

**Comment by wizardk** [comment]
Thanks, it works now.

**Comment by wizardk** [comment]
Thanks, it works now.
